src/python/pants/backend/python/python_requirements.py

```diff
@@ -2,35 +2,62 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 import os
+from typing import Iterable, Mapping, Optional
+
+from pkg_resources import Requirement
 
 
 class PythonRequirements:
-    """Translates a pip requirements file into an equivalent set of python_requirements.
+    """Translates a Pip requirements file into an equivalent set of `python_requirement_library`
```

Comment:
AFAICT, `pip` refers to itself in lowercase: https://pip.pypa.io/en/stable/installing/